### DISCUSSION TITLE
Hospital PTx Standard Schema and Pharmacy PTx Requirement

### DISCUSSION THREAD

**Discussion by santaviccap** [CMSgov/price-transparency-guide#612] [General]
I've heard that there is a standard schema being developed for the Hospital Price Transparency Files.  Also I know that the Pharmacy file has been deferred. 

Is there anybody that has the latest on these two topics?  Do we know when the Hospital standard schema would be implemented.  Do we know if the Pharmacy file is likely to still happen?  @shaselton-usds

**Comment by shaselton-usds** [comment]
Hi @santaviccap -- Thanks for the question. Unfortunately I don't have any information (nor could really comment) on either of those fronts. Just a heads up, this'll probably the answer most of the time when it comes to straight policy questions on Github.

**Reply by dgolden** [reply]
@shaselton-usds Out of curiosity, should any of us want to influence or ask policy questions, what is the point of contact at CMS?  Has there been anything published that describes how policies are formulated? Thanks.

**Reply by JR-CMS** [reply]
The best way to submit policy questions or feedback is through our webform, available on our website here: https://www.cms.gov/healthplan-price-transparency/contact-us

**Comment by shaselton-usds** [comment]
@santaviccap Hi! Revisiting this thread, I wanted to point you to some sample templates that the hospital price transparency team recently made available here: https://www.cms.gov/hospital-price-transparency/resources

There are a couple formats along with a data dictionary.
